django/contrib/admin/static/admin/css/changelists.css

```diff
@@ -148,10 +148,16 @@
     border-bottom: none;
 }
 
-#changelist-filter h3 {
+#changelist-filter details summary {
+    cursor: pointer;
     font-weight: 400;
     padding: 0 15px;
-    margin-bottom: 10px;
+    margin: .6em 0 10px 0;
```

Comment:
And here I re-write the following margin styles https://github.com/django/django/blob/main/django/contrib/admin/static/admin/css/base.css#L550 and https://github.com/django/django/blob/main/django/contrib/admin/static/admin/css/base.css#L134 as I removed the `h3` element